networkit/cpp/centrality/Closeness.h

```diff
@@ -28,11 +30,10 @@ class Closeness : public Centrality {
 	 * @param normalized Set this parameter to <code>false</code> if scores should
 	 * not be normalized into an interval of [0, 1]. Normalization only for
 	 * unweighted graphs.
-	 * @param	checkConnectedness	turn this off if you know the graph is connected
 	 *
 	 */
 	Closeness(const Graph &G, bool normalized = true,
-	          bool checkConnectedness = true);
```

Comment:
Please keep the old constructor so that we do not have a breaking API change. The old constructor can delegate to the new one. Add `checkConnectedness` to the new constructor to make that functionality optional.